src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/ubuntu/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -46,5 +47,27 @@
     </File>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(__BuildType)' == 'Release'">
+    <ArchitectureSpecificNativeSymbol Include="$(BinDir)libcoreclr.so.dbg"/>
+    <ArchitectureSpecificNativeSymbol Include="$(BinDir)libcoreclrtraceptprovider.so.dbg"/>
+    <ArchitectureSpecificNativeSymbol Include="$(BinDir)libdbgshim.so.dbg"/>
+    <ArchitectureSpecificNativeSymbol Include="$(BinDir)libmscordaccore.so.dbg"/>
+    <ArchitectureSpecificNativeSymbol Include="$(BinDir)libmscordbi.so.dbg"/>
+    <ArchitectureSpecificNativeSymbol Include="$(BinDir)libsos.so.dbg"/>
+    <ArchitectureSpecificNativeSymbol Include="$(BinDir)libsosplugin.so.dbg"/>
+    <ArchitectureSpecificNativeSymbol Include="$(BinDir)System.Globalization.Native.so.dbg"/>
```

Comment:
Hmm, do you think it would be worth doing something like this to reduce repetition?

```XML
<NativeSplittableBinary Include="$(BinDir)libcoreclr.so" />
<NativeSplittableBinary Include="$(BinDir)libcoreclrtraceptprovider.so" />

<ArchitectureSpecificNativeFile Include="@(NativeSplittableBinary)" />
<ArchitectureSpecificNativeSymbol Include="@(NativeSplittableBinary -> '%(Identity).pdb')"/>
```